### PR TITLE
inital foundation

### PR DESCRIPTION
**Pull request by jdbbaugh**
1`. git fetch --a `
2. git checkout foundation
3 make sure html looks good

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    <header>
+      <nav>
+          <ul>
+              <li href="">Executive Summary</li>
```
Comment (asiamcarter) [inline index.html]: change to <a href> for links

Comment (asiamcarter) [inline index.html:13]: change to `<a  href>` for links

**Review by asiamcarter — CHANGES_REQUESTED** [review]
looks like you combined `<li>` and `<href>`links should be nested inside of the `<li>`

**Review by ColleenWoolsey — COMMENTED** [review]
looks good

**Review by loetek — APPROVED** [review]
Bueno

**Pull request merged**